build/commands/lib/logging.js

```diff
@@ -1 +1,140 @@
-modules.exports = require('../../../lib/logging.js')
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+const os = require('os')
+const chalk = require('chalk')
+const logUpdate = require('log-update')
+const tsm =
+  process.env.TEAMCITY_VERSION !== undefined
+    ? require('teamcity-service-messages')
+    : undefined
+
+let divider
+function setLineLength () {
+  divider = Array(process.stdout.columns || 32).join('-')
+}
+setLineLength()
+process.stdout.on('resize', setLineLength)
```

Comment:
This seems like an odd place for this. Can you add some comments explaining the purpose and why it's at the top level like this?

Reply:
this file was moved as is from `build/commands/lib/sync/logging.js`, I didn't add this line :).
I guess it just updates "divider" variable when a terminal size changes.